### PR TITLE
issue-3 - example for calling external http during processing http re…

### PR DESCRIPTION
**Pull request by kbertalan**
…sponse

### CODE REVIEW

**Comment by kbertalan** [comment]
An example for #3

File: src/Main.idr [hunk 1]
```diff
@@ -15,15 +15,34 @@ import TyTTP.URL
...
+                      onData res s.onNext
+                      onEnd res s.onSucceded
+                } ctx
           ]
+
+  putStrLn "Calling from main thread"
+  ignore $ http.get "http://localhost:3000/request" $ \res => do
+    putStrLn "response"
+    onData res putStrLn
+    server.close
```
Comment (alexhumphreys) [inline src/Main.idr]: Not sure what you want from this `Main.idr` example, but could be worth removing this `server.close` line so the server stays alive and users can query the server.

Comment (kbertalan) [reply src/Main.idr]: I use this trick to stop the application after I made the test HTTP request, as my usual workflow is to build and run the application on every change to get immediate feedback.

I will remove the whole client call from the example.

Comment (alexhumphreys) [reply src/Main.idr]: Pretty fun trick, I'll start using that!

**Review by alexhumphreys — APPROVED** [review]

**Pull request merged**